src/js/utils/computed-style.js

```diff
@@ -26,7 +26,13 @@ function computedStyle(el, prop) {
   }
 
   if (typeof window.getComputedStyle === 'function') {
-    const computedStyleValue = window.getComputedStyle(el);
+    let computedStyleValue;
+
+    try {
+      computedStyleValue = window.getComputedStyle(el);
+    } catch (_) {
```

Comment:
```suggestion
    } catch (e) {
```
We generally use `e` as the catch placeholder, even when we don't use it directly.

Reply:
yea, makes sense. just updated.